.doc_gen/metadata/cognito-identity-provider_metadata.yaml

```diff
@@ -605,8 +614,9 @@ cognito-identity-provider_Scenario_SignUpUserWithMfa:
           excerpts:
             - description:
               snippet_tags:
-                - cognito.dotnetv3.CognitoBasics.Usings
-                - cognito.dotnetv3.CognitoBasics.Main
+                - Cognito.dotnetv3.Usings
```

Comment:
Should the UIMethods be last in this list? I usually try to put the most relevant code snippets first. Also, I would include a snippet for the wrapper itself here, because without the wrapper they are missing a lot of the necessary code.

Reply:
Moved it to last and also added the CognitoWrapper right after Main.